### PR TITLE
chore: environment variable substitution for the Tauri build

### PR DESCRIPTION
**Pull request by PatrickDinh**
I can't test the Tauri build fully due to there were no code signing certs setup. I can confirm that the build works all the way to right before code signing.

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -84,7 +83,7 @@ jobs:
     strategy:
       matrix:
         # macos-14 is the Apple Silicon M1 runner
-        platform: [ubuntu-20.04, windows-latest, 'macos-12', 'macos-14']
+        platform: [ubuntu-22.04, windows-latest, 'macos-12', 'macos-14']
```
Comment (PatrickDinh) [inline .github/workflows/release.yaml:86]: I had to upgrade to ubuntu 22.04 because libwebkit2gtk-4.1-dev package wasn't found in ubuntu 20.04

Comment (neilcampbell) [reply .github/workflows/release.yaml:86]: Does that alter what ubuntu version we support, assuming going from 20 to 22 results in a different glibc version too. Could an alternative be to install `libwebkit2gtk-4.1` as part of the build process?

Comment (neilcampbell) [reply .github/workflows/release.yaml:86]: Wait I can see that we are installing it. Is it just not compatible with 20?

**Review by neilcampbell — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (PatrickDinh) [reply .github/workflows/release.yaml:86]: yes that is the case. Attempting to install `libwebkit2gtk-4.1` on ubuntu 20.04 resulted in a `the package was not found in registry` error. The issue was mentioned here
https://github.com/tauri-apps/tauri/issues/9662#issuecomment-2411428766
Following that issues on GitHub, it seems to be that at least we need Ubuntu 22.04 to build Tauri v2.
----
I tested with a fresh ubuntu 20.04 VM and can confirm that algokit-lora can't be installed 
image

**Pull request merged**